pkg/generators/validators_info.go

```diff
@@ -54,13 +56,13 @@
 		})
 
 		sort.Slice(activeValidators, func(i, j int) bool {
-			return utils.StrToFloat64(activeValidators[i].DelegatorShares) > utils.StrToFloat64(activeValidators[j].DelegatorShares)
+			return activeValidators[i].DelegatorShares.GT(activeValidators[j].DelegatorShares)
 		})
 
-		var totalStake float64 = 0
+		totalStake := math.LegacyNewDec(0)
 
 		for _, activeValidator := range activeValidators {
-			totalStake += utils.StrToFloat64(activeValidator.DelegatorShares)
+			totalStake = totalStake.Add(activeValidator.DelegatorShares)
```

Comment:
Done, can you check?

Reply:
> Directly check the correctness of the totalStake calculation.

Done here:
```
	totalBondedGauge, ok := results[1].(*prometheus.GaugeVec)
	assert.True(t, ok)
	assert.InEpsilon(t, float64(5), testutil.ToFloat64(totalBondedGauge.With(prometheus.Labels{
		"chain": "chain",
	})), 0.01)
```

> Validate the behavior when DelegatorShares are at their edge values (e.g., very high, very low, zero).
This actually won't do any harm even if there are edge values, as it's just a sum (and it's ok if it's 0).

Reply:
Yeah, but it's just a sum, so shouldn't do any harm here, as it's not divided by 0 or something. This is basically used as a Prometheus metric showing the total staked tokens on chain, so it can be anything.